aws/logs_monitoring/template.yaml

```diff
@@ -154,6 +154,14 @@ Parameters:
       - true
       - false
     Description: Set to true to deploy the Forwarder to a VPC and send logs, metrics, and traces via a proxy. When set to true, must also set VPCSecurityGroupIds and VPCSubnetIds.
+  DdParseLegacyMonitoringLogs:
+    Type: String
+    Default: false
+    AllowedValues:
+      - true
+      - false
+    Description: Set to true to enable the support for converting legacy MONITORING| formatted lines to distribution metrics. 
```

Comment:
Let's also mention "`.dist` will be prepended to metric names to differentiate from the non-distribution ones.